benchmarking/tuning.py

```diff
@@ -32,7 +33,15 @@ def tune(
     Raises:
         ValueError: If no trials are returned by the parallel run of tuning.
     """
-    run = parallel_ex.run(config_updates=parallel_run_config)
+    search_alg = optuna.OptunaSearch()
+    updated_parallel_run_config = copy.deepcopy(parallel_run_config)
```

Comment:
Took me quite some time to understand how you are updating the config here (ok maybe I should have a :coffee: ). But I think this is essentially the same as this?
```python
    if "tune_run_kwargs" in updated_parallel_run_config:
        updated_parallel_run_config["tune_run_kwargs"]["search_alg"] = optuna.OptunaSearch()
    else:
        updated_parallel_run_config["tune_run_kwargs"] = dict(search_alg=optuna.OptunaSearch())
```

Reply:
Done!

Reply:
thx!